second-readme.md

```diff
@@ -0,0 +1,15 @@
+# Preparation work
+
+Congratulations, you've made it through the [first round of preparation](README.md)! You're up to the next stage. 
+
+Complete these steps **before** you begin Phase 0.
+
+1. [Our expectations](/expectations/) (5-20 minutes)
+2. [Best practices for success](/best-practices/) (10 minutes)
+3. [Installation](/installation) (varies, start now)
+4. [Tools you will use](/tools/) (45 minutes)
```

Comment:
There is no way someone can read all of that content (let alone understand it all) in 45 minutes. I also think it should be moved into the next section.